### DISCUSSION TITLE
During Entity Edit, allow user to tab between multiple parameter values - e.g., X/Y for point

### DISCUSSION THREAD

**Discussion by dtremain** [hlorus/CAD_Sketcher#467] [Feature Requests]
When you create a rectangle or a point, if you enter the parameters at the time of creation, you can tab through them. If you do it though the entities edit, you have to click on one value to be able to change it, Enter to accept the change, then click on the next parameter to change, etc.

**Comment by Chrissie-AI** [comment]
I`ll second that. This is such an obvious limitation that I'm amazed noone has come up with that suggestion before.
